data/json/items/magazine/45.json

```diff
@@ -38,8 +38,16 @@
     "id": "tdi_mag",
     "looks_like": "mp5mag",
     "type": "MAGAZINE",
-    "name": { "str": "Vector SMG 30-round magazine" },
-    "description": "A 30-round polymer and steel box magazine for use with the KRISS Vector.",
+    "name": { "str": "Glock SMG 30-round magazine" },
+    "description": "A 30-round polymer and steel box magazine for use with the Glock submachine gun.",
```

Comment:
Glock doesn't make submachine guns.

Reply:
The Vector is an SMG taking Glock magazines (and it's own special one). I think it's more clear to call it a Glock SMG and call it's magazine a Glock SMG magazine, than to be accurate.

That is, unless there is some sort of change that could come in the future that will change it from accepting only it's own magazines (that go into no other guns), and Glock magazines, or Glock decides to release an SMG of its own that does not accept Vector magazines.